src/store/education/slice.ts

```diff
@@ -0,0 +1,25 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+import { IEducation } from 'src/common/types';
+import { addEducation } from './actions';
+
+interface IEducationState {
+  education: IEducation[];
+}
+
+const initialState: IEducationState = {
+  education: [],
+};
+
+const { reducer, actions } = createSlice({
+  name: 'education',
+  initialState,
+  reducers: {},
+  extraReducers: builder => {
+    builder.addCase(addEducation.fulfilled, (state, { payload }) => {
+      state.education.push(payload);
```

Comment:
Are we going to have 'education' inside the 'education' slice? Let's rename it to something proper